app/assets/stylesheets/views/_transition-landing-page.scss

```diff
@@ -42,13 +42,12 @@ $header-breakpoint: 960px;
   @include govuk-media-query($from: $header-breakpoint) {
     padding-bottom: 80px;
     padding-top: govuk-spacing(5);
-    background: linear-gradient(
-      to right, 
-      $dark-blue 0%, 
-      $dark-blue 66.66%, 
-      $transparent 66.66%, 
-      $transparent 100%
-    );
+
+    background: $dark-blue;
+    background: -moz-linear-gradient(90deg, $dark-blue 66.66%, $transparent 66.66%);
+    background: -webkit-linear-gradient(90deg, $dark-blue 66.66%, $transparent 66.66%);
+    background: linear-gradient(90deg, $dark-blue 66.66%, $transparent 66.66%);
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#{$dark-blue}", endColorstr="#{$transparent}", GradientType=1);
```

Comment:
I don't think that this filter creates a gradient in the same way that the CSS gradient does - it just gradually goes from blue to transparent. It could be worth removing, since the solid colour background would create a decent enough fallback.

Reply:
I might just remove the image for IE9 and below

Reply:
Hey - remove this and removed the image completely on IE9 and below